### PR TITLE
Fix in-app documentation

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

https://github.com/gravitee-io/issues/issues/7737

**Description**

The use of outdated dependencies for the in-app documentation was causing some errors, so it has been decided to move to `marked` + `highlight.js`. I also fixed a couple of documentation pages because they either contained typos or weren't "correct" markdown.

**Screenshots**

With tables and titles:
image

With huge content and code:
image

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/7737-replace-showdown-by-marked/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

File: gravitee-apim-console-webui/docs/management-application-subscriptions.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 Retrieve all the registered subscriptions per API. Subscription represents the plans the application has subscribed to.
 
-Subscriptions can be : 
+Subscriptions can be :
```
Comment (phiz71) [inline gravitee-apim-console-webui/docs/management-application-subscriptions.md]:
```suggestion
Subscriptions can be:
```

File: gravitee-apim-console-webui/docs/management-configuration-apiportalheader.md [hunk 2]
```diff
@@ -5,15 +5,15 @@ The name could be used as is, or could be a translated key.
...
-api.version | `${api.version}`
-api.endpoint | `${api.proxy.contextPath}`
-api.publishedAt | `${(api.deployedAt?date)!}`
-My hard coded name | `My hard coded value`
+| Name               | Value                       |
+|--------------------|-----------------------------|
+| api.version        | `${api.version}`            |
+| api.endpoint       | `${api.proxy.contextPath}`  |
+| api.publishedAt    | `${(api.deployedAt?date)!}` |
+| My hard coded name | `My hard coded value`       |
```
Comment (phiz71) [inline gravitee-apim-console-webui/docs/management-configuration-apiportalheader.md:17]: 😍

File: gravitee-apim-console-webui/docs/management-configuration-categories.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Portal categories
 
-Categories let you organize automatically into categories the APIs. 
+Categories let you organize automatically into categories the APIs.
 Categories allow to filter, sort, or group API to make them easier to find by the consumers.
 
-Categories are visible into the APIs portal page only if one or more API belong to these categories.
+Categories are visible into the APIs portal page only if one or more API belongs to these categories.
```
Comment (phiz71) [inline gravitee-apim-console-webui/docs/management-configuration-categories.md]: "one or more" is singular or plural ?

File: gravitee-apim-console-webui/docs/management-configuration-top_apis.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Portal top APIs
 
-Top APIs are visible into the APIs portal home page only if one or more API belong to these categories.
+Top APIs are visible into the APIs portal home page only if one or more API belongs to these categories.
```
Comment (phiz71) [inline gravitee-apim-console-webui/docs/management-configuration-top_apis.md]: "one or more" is singular or plural ?

File: gravitee-apim-console-webui/docs/management-application-subscriptions.md (hunk 1 above)
Comment (gaetanmaisse) [reply gravitee-apim-console-webui/docs/management-application-subscriptions.md]: Nice catch! I missed some of these, will do a 2nd find and replace

File: gravitee-apim-console-webui/docs/organization-configuration-roles.md [hunk 5]
```diff
@@ -4,23 +4,23 @@ A role can be scoped to an API, an application, the portal or more globally to t
 
 Each role defines a set of permissions (READ, CREATE, UPDATE, DELETE (CRUD)) on the API Management resources.
 
-An role can be edited by clicking on the *edit* button on each role scope :
+A role can be edited by clicking on the *edit* button on each role scope :
```
Comment (phiz71) [inline gravitee-apim-console-webui/docs/organization-configuration-roles.md]:
```suggestion
A role can be edited by clicking on the *edit* button on each role scope:
```

File: gravitee-apim-console-webui/docs/management-configuration-categories.md (hunk 3 above)
Comment (gaetanmaisse) [reply gravitee-apim-console-webui/docs/management-configuration-categories.md]: It looks like the correct one is `Categories are visible on the APIs portal page only if one or more APIs belong to these categories.`

**Review by phiz71 — APPROVED** [review]

**Pull request merged**